lib/components/navigation/DashboardLink.tsx

```diff
@@ -34,7 +36,7 @@ const DashboardLink: React.FC<DashboardLinkProps> = ({
   return (
     <Link href={href} passHref>
       <a
-        className={`${borderColor} flex h-48 w-72 flex-1 flex-row justify-center rounded-2xl border bg-ocf-black-500 p-5 text-center md:text-left`}
+        className={`${borderColor} flex h-fit w-72 flex-1 flex-row justify-center rounded-2xl border bg-ocf-black-500 p-5 text-center md:text-left`}
```

Comment:
```suggestion
        className={`${borderColor} flex h-fit flex-1 flex-row justify-center rounded-2xl border bg-ocf-black-500 p-5 text-center md:text-left`}
```
this will make everything work with the additional changes I suggested below.